tf/tfprocess.py

```diff
@@ -66,6 +66,14 @@ def __init__(self, cfg):
         # For exporting
         self.weights = []
 
+        self.swa_enabled = self.cfg['training']['swa']
+
+        # Limit momentum of SWA exponential average to 1 - 1/swa_max_n
+        self.swa_max_n = self.cfg['training']['swa_max_n']
+
+        # Net sampling rate (e.g 2 == every 2nd network).
+        self.swa_c = self.cfg['training']['swa_steps']
```

Comment:
This seems unused.

Reply:
SWA skip has always been 1 in LZ as far as I know. I think it's pretty safe to remove this and assume that every test evaluation network is averaged.